packages/fuselage/src/components/Box/Scrollable/index.js

```diff
@@ -1,9 +1,24 @@
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { useCallback } from 'react';
 
 import { useProps } from '../../../hooks';
 
-export function Scrollable({ children, horizontal, vertical, smooth }) {
+export function Scrollable({ children, horizontal, vertical, smooth, scrollCb }) {
+  let scrollTimeout;
+
+  const handleScroll = useCallback(function(event) {
+    const { target } = event;
+    if (!scrollTimeout) {
+      scrollCb({ top: !!target.scrollTop, bottom: !!(target.scrollTop + target.clientHeight - target.scrollHeight), left: !!target.scrollLeft, right: !!(target.scrollLeft + target.clientWidth - target.scrollWidth) });
+    } else {
+      clearTimeout(scrollTimeout);
+    }
+    scrollTimeout = setTimeout(() => {
+      scrollTimeout = false;
+      scrollCb({ top: !!target.scrollTop, bottom: !!(target.scrollTop + target.clientHeight - target.scrollHeight), left: !!target.scrollLeft, right: !!(target.scrollLeft + target.clientWidth - target.scrollWidth) });
+    }, 200);
```

Comment:
What about `scrollTimeout === undefined ? 0 : 200`? That if would become superfluous and some code duplication will vanish.